src/main.ts

```
@@ -54,6 +54,7 @@ async function getDiff(
   });
   // @ts-expect-error - response.data is a string
   return response.data;
+  return response;
```

Biome

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)